Mathlib/Topology/ContinuousMap/Compact.lean

```diff
@@ -337,6 +337,12 @@ theorem linearIsometryBoundedOfCompact_of_compact_toEquiv :
 
 end
 
+@[simp] lemma norm_smul_const {R β : Type*} [NormedAddCommGroup β] [NormedDivisionRing R]
```

Comment:
Please prove a `nnnorm` version. If you prove it first, then this one is `mod_cast nnnorm_smul_const f b`.